test_configs/sockets/quick.test

```diff
@@ -70,7 +70,7 @@
 {
 	prov_name: sockets,
 	test_type: [
-		FT_TEST_LATENCY,
+		FT_TEST_LATENCY
```

Comment:
I think there is no need to go out of the way to add the missing commas. It can probably be added whenever we find one missing. :)